pkg/agent/controller/networkpolicy/priority.go

```diff
@@ -144,6 +144,17 @@ func (pa *priorityAssigner) GetOFPriority(p types.Priority) (*uint16, map[uint16
 	return &ofPriority, map[uint16]uint16{}, nil
 }
 
+// RegisterPriorities registers a list of Priorities to be created with priorityMap.
+// It is used to populate the priorityMap in case of batch rule adds.
+func (pa *priorityAssigner) RegisterPriorities(priorities []types.Priority) error {
+	for _, p := range priorities {
+		if _, _, err := pa.GetOFPriority(p); err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
Yes. The optimization is mostly done in the fact that the flows will be installed in a single bundle now. Priority-wise, time complexity is relatively harder to optimize and would not gain us much, since the procedures are all just map manipulations and are fairly quick.